declarations/Finya.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Finya",
+  "documents": {
+    "Terms of Service": {
+      "fetch": "https://www.finya.de/Legal/terms/",
+      "select": [
+        "main"
+      ]
+    },
+    "Privacy Policy": {
+      "fetch": "https://www.finya.de/Legal/privacy/",
+      "select": [
+        "main"
+      ],
+      "remove": [
+        "main > div:last-child"
+      ]
```

Comment:
The registered version will be cleaner without the div that contains the links related to this page 👍  but the remove selector seems to me to introduce a risk of deleting content that could indicate future modifications.

What do you think?

Reply:
I agree that risk is there. If including the links is fine maybe we can leave them in?

Reply:
As we don't have a stable way of removing them, I think we can keep them 👍 Could you include this modification?

Reply:
```suggestion
```